sdk/go/provision_test.go

```diff
@@ -52,20 +53,23 @@ func TestImageProvision(t *testing.T) {
 		t.Fatalf("failed to create container: %s", output)
 	}
 
-	parallelism := 30
+	parallelism := 5
```

Comment:
We should set this based on the number of available CPUs, rather than hard-code this to an arbitrary value. Pretty sure it's going to be more wrong if we go for a hard-coded value, rather than basing it on the number of CPUs.